### PR TITLE
New package: skk-jisyo-20181111

### PR DESCRIPTION
**Pull request by rail44**
SKK is one of the input method for Japanese.
Though there are implemlementions such as fcitx-skk or ibus-skk, every use dictionary files called skk-jisyo on many distributions.

### CODE REVIEW

File: srcpkgs/skk-jisyo/template [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Template file for 'skk-jisyo'
+pkgname=skk-jisyo
+version=20181111
+revision=1
+noarch=yes
+build_style=fetch
+short_desc="Dictionaries for the SKK Japanese input method"
+maintainer="Satoshi Amemiya <amemiya@protonmail.com>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/skk-jisyo/template]: please use SPDX identifier

File: srcpkgs/skk-jisyo/template [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+homepage=http://openlab.ring.gr.jp/skk/
```
Comment (maxice8) [inline srcpkgs/skk-jisyo/template]: please quote homepage

File: srcpkgs/skk-jisyo/template [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+ 851b076be6ca24bea975f986f7386bada33128014e024ae7adfef0bd2131cba3
+ 00d951a4c1f4ac9441e4b10c3f4a15dfcc8b1e4b54d21a75f0be575fdedff6a6
+ 4634b29252e0c7ee199447cf2acfea2cbec13a4f1fcc9d3876feb3a72e7590e4
+ dfa8b678c1bbc8edb3df768d21359efd4e8c3bf6bd3a27987b39966f7f6e0408
+ 15b169e489a014969d701f4de93ee21867a6d71a6df5381990e4631c4fb778c1
+ 2000415c584607e0d23d8d83ff448a0eb294b5d03c35cb0282b064531e51de9b
+ ad57ab4608b5aca8377a2c2c43980c196eb2d514f9dcfdc1fd3f880d22dab33c"
+
+pre_install() {
+	gunzip SKK-JISYO.*.gz
```
Comment (maxice8) [inline srcpkgs/skk-jisyo/template]: no need to have this in pre_install() if you are defining do_install()

**Comment by rail44** [comment]
fixed up!

**Comment by maxice8** [comment]
The distfiles links are not versioned, when they change it will break the package.

Aren't there versioned distfiles available ?

**Comment by rail44** [comment]
@maxice8 
Unfortunately, there is no versioned one.
Because the dictionary is plain text, many distributions package it with date, like some icon packs or fonts.

**Comment by maxice8** [comment]
Yeah but without using a dated archive they will change the checksum wildly.

We absolutely need a versioned archive

**Comment by rail44** [comment]
@maxice8 
hmm, is it acceptable that fetching the archive from repository of other distribution?
is there any package choosing such way on void?

**Comment by rail44** [comment]
sorry, I changed my mind. I've choosed to run script for feting dictionary 👍

**Comment by rail44** [comment]
@maxice8 
thank you for your review 😄

**Pull request closed without merging**